src/transformers/modeling_albert.py

```diff
@@ -334,9 +332,6 @@ def forward(self, hidden_states, attention_mask=None, head_mask=None):
             # Index of the hidden group
             group_idx = int(i / (self.config.num_hidden_layers / self.config.num_hidden_groups))
 
-            # Index of the layer inside the group
-            layer_idx = int(i - group_idx * layers_per_group)
```

Comment:
cc @LysandreJik please check that this `layer_idx` was indeed dead-code and something that should have been used in the subsequent lines